core/extension/extension_api_dump.cpp

```diff
@@ -70,7 +78,7 @@ static String get_type_name(const PropertyInfo &p_info) {
 	if (p_info.type == Variant::NIL) {
 		return "void";
 	}
-	return Variant::get_type_name(p_info.type);
+	return get_builtin_or_variant_type_name(p_info.type);
```

Comment:
I don't think that's correct as is, see the code just above. `Variant::NIL` can be either `void`/`Nil`, or `Variant` if `PROPERTY_USAGE_NIL_IS_VARIANT` is defined. So you need to check that and set the name accordingly IMO.

Reply:
Actually the code above already handles both cases so the new method would never actually get called with `Variant::NIL` so it's probably fine. But if `usage` is available in the other places, it should be handled to be accurate.